frontend/src/components/volunteer/Volunteer.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Row } from 'react-bootstrap';
+import styles from "./volunteer.module.css";
+import Container from 'react-bootstrap/Container';
+import Col from 'react-bootstrap/Col';
+var ScrollArea = require('react-scrollbar');
```

Comment:
Match the styles above. We are using ES6 so we don't need require here. Use import instead.